spec/controllers/hyrax/depositors_controller_spec.rb

```diff
@@ -62,6 +62,16 @@
           expect { redundant_request_to_grant_proxy }.not_to change { user.mailbox.inbox.count }
         end
       end
+
+      context "when a request to grant proxy is made to a user" do
+        let(:request_to_grant_proxy) { post :create, params: { user_id: user.user_key, grantee_id: grantee.user_key, format: 'json', use_route: :hyrax } }
+        let(:expected_delete_path) { "/users/" + user.user_key.gsub("\.", '-dot-') + "/depositors/" + grantee.user_key.gsub("\.", '-dot-') + "?locale=en" }
+
+        it "the user is made proxy, and the delete proxy button path is valid" do
```

Comment:
This test appears to be testing the change that makes the feature possible, and not necessarily the feature.  Please rework to test the feature